src/pages/documentation/identity-for-edge-network/faq.md

```diff
@@ -141,3 +141,11 @@ iOS
 ## Q: Can I safely remove the Identity for Experience Cloud ID Service extension in an app if I am using the Edge Network extension?
 
 If no other extension relies on the Identity for Experience Cloud ID Service extension, you can safely remove it. Please refer to the [Identity consideration document](../adobe-analytics/migrate-to-edge-network.md#identity-consideration-for-the-migration) for more information.
+
+## Q: In my hybrid app, I need to pass the mobile SDK's ECID to the WebView using getUrlVariables API. However, the Web SDK generates a new ECID instead of recognizing the existing one. How to fix this?
+
+1. Ensure that the output from the getUrlVariables API includes accurate ECID and OrgID values.
+2. The timestamp (TS) included in the query string from the getUrlVariables API indicates its generation time, and its validity is limited to a 5-minute window from that moment.
+3. The query string produced by the getUrlVariables API is already encoded, so please censure that you do not re-encode the generated query string.
+
+For further information, refer to the documentation on [Mobile-to-web and cross-domain ID sharing](https://experienceleague.adobe.com/docs/experience-platform/edge/identity/id-sharing.html?lang=en) and [Managing Webviews](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/app-implementation/web-views.html?lang=en#implementation).
```

Comment:
title/link updates:
[mobile-to-web and cross-domain ID sharing](https://experienceleague.adobe.com/docs/experience-platform/edge/identity/id-sharing.html) and [handling WebViews](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/app-implementation/web-views.html)